code/modules/jobs/job_types/medical_doctor.dm

```diff
@@ -21,6 +21,16 @@
 
 	family_heirlooms = list(/obj/item/storage/firstaid/ancient/heirloom)
 
+	mail_goodies = list(
+		/obj/item/healthanalyzer/advanced = 15,
+		/obj/item/scalpel/advanced = 6,
+		/obj/item/retractor/advanced = 6,
+		/obj/item/cautery/advanced = 6,
+		/datum/reagent/toxin/formaldehyde = 6,
+		/obj/effect/spawner/lootdrop/organ_spawner = 5,
+		/obj/effect/spawner/lootdrop/memeorgans = 1
```

Comment:
you might wanna add abductor organs to the pool here as well

Reply:
Covered on line 30 already.